New Relic Synthetics quota (#8956)
pantheon-systems/documentation: source/content/guides/new-relic/03-monitor-new-relic.md

- [Troubleshooting WordPress Performance with New Relic&reg; Performance Monitoring](https://pantheon.io/blog/troubleshooting-wordpress-performance-new-relic)

## Configure Ping Monitors (Synthetics) for Availability

New Relic provides an availability monitoring service within their Synthetics tool suite which can be enabled by [contacting support](/guides/support/contact-support/). This basic monitoring feature sends a request to designated URLs from configured locations. Requests are sent at configured intervals and alerts are sent via email when a response fails.

> Pantheon can provide New Relic Synthetics ping monitoring as part of the service with a usage quota limited to 10K Synthetics checks per month for each Pantheon workspace.

Reviewer: ```suggestion
Pantheon caps usage of ping monitoring at 10K Synthetics checks per month for each Pantheon workspace.
```

Author: @stevector We need to clarify this: the quota doesn't include ping monitors (customers can use unlimited ping monitors); the limit only applies to non ping monitors. I'd propose something like this:
```suggestion
Pantheon provides unlimited New Relic Synthetics ping monitoring as part of the service. Still, there's a quota for all other monitor types (simple browser, scripted browser, and scripted API monitors). This quota is set at 10K Synthetics monthly checks for each Pantheon account, so it's crucial to manage it effectively.
```

<Alert title="Note" type="info">

> When a Pantheon workspace has more than 10K Synthetics checks are running monthly Pantheon will adjust the frequency or number of locations a Synthetics check originates from the reduce the number of checks. This is required to allow Synthetic to continue to operate correctly under the overall New Relic quota for Pantheon.

Reviewer: @tyherbert will our customers know that/when this reduction happens?

Author: Yes, we plan to notify the customer via email if a change has been made to their Synthetics monitors.

Reviewer: So is the flow something like
* Pantheon staff somehow sees that a workspace has exceed 10K
* Pantheon staff manually adjust settings in New Relic
* Pantheon staff manually emails customer?

Who from the workspace will get that email?

Author: Correct. This is something that Manoj and I will be working on. We are following the customer communication flow (https://docs.google.com/document/d/1Myagjy-2JhfnG7AqLs6clQ2YV0jxRvz_6nxQK898RQg/edit#heading=h.2licx9cbwmja). Currently we send an email to the site owner and cc the admins (this is a process that Shaq and Manoj put together).
